src/main/webapp/js/build-message-div.js

```diff
@@ -27,4 +27,61 @@ function buildMessageDiv(message){
   messageDiv.appendChild(bodyDiv);
 
   return messageDiv;
+}
+
+function buildMessageInTimeline(message, flip){
+  const imageDiv = document.createElement('div');
+  imageDiv.classList.add("timeline-image");
+
+  const image = document.createElement('img');
+  image.classList.add("rounded-circle");
+  image.classList.add("img-fluid");
+  image.src ="/img/products-01.jpg";
```

Comment:
What would the plan be for this in the future? Would it be the user profile pic or a picture that they upload?

Also note that the picture should be cropped to a square for it to show properly.

Reply:
I was thinking about the picture that the user uploads, but as they are embedded in the message it may not be a good choice. Maybe for now we just plan to add profile pic?